frontend/ember-cli-build.js

```diff
@@ -9,11 +9,15 @@ module.exports = function(defaults) {
   });
   app.import('vendor/js/jquery.js')
   app.import('vendor/js/jquery.tmpl.js')
-  app.import('vendor/js/bootstrap.min.js')
+
+  app.import('bower_components/jquery/dist/jquery.min.js');
```

Comment:
removed.
struggling with displaying a list of scripts returned via the API

routes/user-home.js
```
export default Ember.Route.extend({
model(){
  return Ember.$.get('/getScriptList', 'json'); // returns for now -> [{"filename":"somename","Date":"Date string","Id":1}]
}
});
```
templates/user-home.hbs
```
	{{model}}
	<ul>
	{{#each model as |key value|}}
	  <li>{{key}}: {{value}}</li>
	  {{else}}
	    Sorry, no scripts
	{{/each}}
	</ul>
```
output
```
[{"Date": "Sat, 29 Jul 2017 18:08:46 GMT", "Filename": "asd", "Id": 1}]
Sorry, no scripts
```
The model seems to be fetched correctly (even model.[0] is shown correctly), but I can't iterate through it. Any ideas?